css/templates/lune/_type.css

```diff
@@ -0,0 +1,31 @@
+@import 'page-vars';
+
+/* Bass CSS responsive typescale does not provide h0 and h00
+  below 640px screens. So we defined them here.
+  TODO: Raise issue with base css to get this fixed. */
+
+.h00 {
+  font-size: var(--h00);
+}
+
+.h0 {
+  font-size: var(--h0);
+}
+
```

Comment:
Responsive typescale defines sm-h0, sm-h00, but does not define .h0 for viewports smaller than 640px. So we've added them in here